tools/run_tests/performance/scenario_config.py

```diff
@@ -1659,33 +1659,23 @@ def __str__(self):
 
 
 class NodeLanguage(Language):
-    def __init__(self, node_purejs=False):
-        super().__init__()
-        self.node_purejs = node_purejs
-
     def worker_cmdline(self):
-        fixture = "native_js" if self.node_purejs else "native_native"
+        fixture = "native_js"
```

Comment:
This `fixture` should be set to `js_js`.

Reply:
Done.